js_modules/dagster-ui/packages/ui-core/src/app/DefaultFeatureFlags.oss.tsx

```diff
@@ -5,12 +5,9 @@ import {FeatureFlag} from 'shared/app/FeatureFlags.oss';
  */
 export const DEFAULT_FEATURE_FLAG_VALUES: Partial<Record<FeatureFlag, boolean>> = {
   [FeatureFlag.flagAssetSelectionWorker]: true,
-  [FeatureFlag.flagAssetSelectionSyntax]: new URLSearchParams(global?.location?.search ?? '').has(
+  [FeatureFlag.flagSelectionSyntax]: new URLSearchParams(global?.location?.search ?? '').has(
     'new-asset-selection-syntax',
```

Comment:
I'll keep the URL param the same only because I've already handed it out to a few people and don't want to create any thrash for them.